data/sql/updates/pending_db_world/rev_1609109042026647800.sql

```diff
@@ -0,0 +1,368 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1609109042026647800');
+
+-- ----------------------------
+-- Table structure for gtoctclasscombatratingscalar_dbc
+-- ----------------------------
+DROP TABLE IF EXISTS `gtoctclasscombatratingscalar_dbc`;
+CREATE TABLE `gtoctclasscombatratingscalar_dbc`  (
+  `ID` int(11) NOT NULL DEFAULT 0,
+  `Data` float NOT NULL DEFAULT 0,
+  PRIMARY KEY (`ID`) USING BTREE
+) ENGINE = MyISAM CHARACTER SET = utf8 COLLATE = utf8_general_ci ROW_FORMAT = Fixed;
+
```

Comment:
1) wasn't the table already existing? why not truncate it instead?
@Winfidonarleyan @FrancescoBorzi 

2) it's the issue reported by Wotex?